### DISCUSSION TITLE
Ability to reset tracks without reboot

### DISCUSSION THREAD

**Discussion by flygfantast** [jvde-github/AIS-catcher#277] [Ideas]
A "Reset track" and/or "Reset all tracks" button would be a nice feature. I have many local boats that never turn off the AIS transponder and also some SAR helicopters so the tracks get a bit cluttered after a while.

**Comment by jvde-github** [comment]
It needs to be solved on the front end (for folks sharing their webviewer externally). Let me have a look how this can be done.
